.vscode/launch.json

```
@@ -0,0 +1,15 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
+  // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Python: Main Window",
+      "type": "python",
+      "request": "launch",
+      "program": ".",
```

```suggestion
      "module": "kamyroll_gui",
```
Since we create a `__main__.py` use `"module": "kamyroll_gui"` here

---

I created `__main__.py` to avoid calling as a module since you didn't include a `__main__.py` in the `kamyroll_gui` directory. The current file is outside the module. But I understand what you're saying.